### ISSUE TITLE
[Copy Improvements] Review and update copy with the rafiki test wallet

### ISSUE THREAD

**Issue by RabebOthmani** [interledger/web-monetization-extension#577]
### Context

When performing certain tasks within the extension, the user might be redirected to a web page to get a confirmation or a message from the test wallet. Some of the messages can be improved to be clerer 

### Todos

- Update message 'your request was successfully rejected'

**Comment by melissahenderson** [comment]
Do you have a specific example of what can cause this message? If it happens for a variety of reasons, it becomes more difficult to create helpful messaging.

**Comment by RabebOthmani** [comment]
This happens when you basically don't agree to the grant. I find the words successfully rejected together weird plus , it is me rejecting my own grant which makes it also unusual and weird

**Comment by sidvishnoi** [comment]
"your request was successfully rejected" message is part of our /welcome screen, not test wallet. Should I move this issue to WICG repo which hosts the /welcome page?

**Comment by melissahenderson** [comment]
@sidvishnoi Yeah, I think so. That way we have the background for why that file was changed. 

@RabebOthmani Ideally, a message like this would describe what happened (request canceled) and what the user should do next. To try the request again, would the user need to click the connect button in the extension again?

**Comment by RabebOthmani** [comment]
Issue is now here https://github.com/WICG/webmonetization/issues/497
